File: sample/src/main/java/com/alfianyusufabdullah/sample/fragment/PageThree.java
package com.alfianyusufabdullah.sample.fragment;


import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.alfianyusufabdullah.sample.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class PageThree extends Fragment {

    public PageThree() {
    }

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_page_three, container, false);
    }

}

File: sample/src/main/java/com/alfianyusufabdullah/sample/fragment/PageThree.kt
package com.alfianyusufabdullah.sample.fragment


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.alfianyusufabdullah.sample.R

/**
 * A simple [Fragment] subclass.
 */
class PageThree : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_page_three, container, false)
    }

}
